rotaract3142/www/navbar.html

```diff
@@ -24,10 +24,13 @@
 						<a href="javascript:void(0)"></a>
 					</li>
 					<li>
-						<a href="assembly">District Assembly & AARAs</a>
+						<a href="assembly">District Assembly & AARAs 2020</a>
 					</li>
 					<li>
 						<a href="https://www.rotaract3142.kanr.is/next">NEXT-The Conclave 2020</a>
+					</li>
+					<li>
+						<a href="explore3142">Explore 3142 - 2020</a>
```

Comment:
where is the explore3142 page? This won't work @smitmirani

Reply:
explore3142.html is in the www folder. same as the home page and assembly page.